test/recipes/15-test_gendsa.t

```diff
@@ -107,6 +107,58 @@ ok(!run(app([ 'openssl', 'genpkey',
               '-algorithm', 'DSA'])),
    "genpkey DSA with no params should fail");
 
+
+# test key generation with dsaparam tool
+ok(run(app([ 'openssl', 'dsaparam',
+             '-genkey',
+             '-text',
+             '1024',
+             ])),
+   "dsaparam -genkey DSA 1024 with default qbits");
+
+ok(run(app([ 'openssl', 'dsaparam',
+             '-genkey',
+             '-text',
+             '2048',
+             ])),
+   "dsaparam -genkey DSA 2048 with default qbits");
+
+ok(run(app([ 'openssl', 'dsaparam',
+             '-genkey',
+             '-text',
+             '3072',
+             ])),
+   "dsaparam -genkey DSA 10307224 with default qbits");
+
+ok(run(app([ 'openssl', 'dsaparam',
+             '-genkey',
+             '-text',
+             '1024', '160',
```

Comment:
Is it possible to verify that the extra argument stuck?

Reply:
Unfortunately, the numqbits is not printed in "-text" output today. So in order to verify it, the "-text" output needs to be changed and it might have much bigger consequences that I am not sure I can predict.

Note that the code above does something similar ('-pkeyopt', 'qbits:256') and it doesn't seem it knows how to verify it.